recipes/example/meta.yaml

```diff
@@ -34,9 +33,10 @@ build:
 
 requirements:
   build:
+    - {{ compiler('c') }}
```

Comment:
It's needed in this case as the `example` (`simplejson`) does compile code. Was an oversight that we didn't include `toolchain` before. That said, wouldn't think this would be necessary for things that don't compile code.